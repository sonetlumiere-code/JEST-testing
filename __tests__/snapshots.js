const client = {
    name: 'John',
    balance: 500,
    type: 'Premium'
};

describe('Testing to client', () => {
    test('Client is John', () => {
        expect(client).toMatchSnapshot();
    });
});
