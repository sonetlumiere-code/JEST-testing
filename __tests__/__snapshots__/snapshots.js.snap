// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing to client Client is John 1`] = `
Object {
  "balance": 500,
  "name": "John",
  "type": "Premium",
}
`;
